### PR TITLE
ZIOS-9273 update overlayWindow.frame when applicationWillEnterForeground instea…

### PR DESCRIPTION
**Pull request by billypchan**
This PR is a follow up of https://github.com/wireapp/wire-ios/pull/1437
Fix the overlayWindow not cover the whole screen issue after device rotated.

### CODE REVIEW

File: Wire-iOS/Sources/AppRootViewController.swift [hunk 1]
```diff
@@ -56,7 +56,10 @@ class AppRootViewController : UIViewController {
 
     override public func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         mainWindow.frame.size = size
-        overlayWindow.frame.size = size
+
+        coordinator.animate(alongsideTransition: nil, completion: { _ in
+            self.overlayWindow.frame = UIScreen.main.bounds
```
Comment (typfel) [inline Wire-iOS/Sources/AppRootViewController.swift]: Why is this necessary and won't behave badly on iPad when the app is in split screen mode?

**Comment by billypchan** [comment]
@typfel @mikeger rewrite overlayWindow's frame update method with your comments: 

````
    func updateOverlayWindowFrame() {
        self.overlayWindow.frame = UIApplication.shared.keyWindow?.frame ?? UIScreen.main.bounds
    }
````

**Comment by typfel** [comment]
@billypchan do we have any steps-to-reproduce for the issue that this fixes?

**Comment by billypchan** [comment]
@typfel I recorded a video in ticket ZIOS-9273.
